src/deluge/gui/views/view.cpp

```diff
@@ -1122,6 +1122,11 @@ void View::setModLedStates() {
 				}
 			}
 		}
```

Comment:
Added to restore clip button blinking when in keyboard screen and automation instrument clip view is opened in the background